verifier/providerverifier.go

```diff
@@ -241,3 +300,13 @@ func verifyAudience(pkt *pktoken.PKToken, clientID string) error {
 	}
 	return nil
 }
+
+func VerifyCicSignature(pkt *pktoken.PKToken) error {
```

Comment:
```suggestion
func verifyCicSignature(pkt *pktoken.PKToken) error {
```

Should this be private since it is important that the token's signature has been verified before the CIC is verified? So it should only ever be called from `VerifyProvider` after the pkt signature is verified?